src/rgw/rgw_aio.h

```diff
@@ -73,7 +73,7 @@ inline int check_for_errors(const AioResultList& results) {
 // each call returns a list of results from prior completions
 class Aio {
  public:
-  using OpFunc = fu2::unique_function<void(Aio*, AioResult&) &&>;
+  using OpFunc = fu2::unique_function<void(Aio*, AioResult&)>;
```

Comment:
This is part of the contract for the interface, specifically a function we will:
1. Never copy (thus it doesn't need to be copyable)
2. Only call once. That's basically the point of having the call operator rvalue qualified.